src/components/Images.vue

```diff
@@ -0,0 +1,205 @@
+<!--
+  - This is the Images table in the K8s page.
+  -->
+<template>
+  <div>
+    <div v-if="isRunning">
+      <Checkbox
+        :label="'Show all images'"
+        :value="showAll"
+        @input="handleCheckbox"
+      />
+      <SortableTable
+        :headers="headers"
+        :rows="rows"
+        key-field="key"
+        default-sort-by="imageName"
+        :table-actions="false"
+        :paging="true"
+      >
+        <template #row-actions="{row}">
+          <div v-if="hasDropdownActions(row)">
+            <ButtonDropdown
+              :button-label="'...'"
+              :dropdown-options="buttonOptions(row)"
+              size="sm"
+              @click-action="(rowOption) => doClick(row, rowOption)"
+            />
+          </div>
+          <div v-else></div>
```

Comment:
Yes we do. I don't know why, but without it we get an empty dropdown button. With it no button is rendered at that position.

Reply:
Ah, probably because the `<template>` must have a single child element.  We would probably need something like:

```vue
<template #row-actions="{row}">
  <div>
    <ButtonDropdown
      v-if="hasDropdownActions(row)"
      ⋯
  </div>
</template>
```